app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -387,8 +359,9 @@ private void initSpinner() {
             public void onItemSelected(final AdapterView<?> parent, final View view, final int position, final long id) {
 
                 Chapter selectedChapter = mChapterAdapter.getItem(position);
-                updateSelectionfor(selectedChapter);
+                updateSelectionfor(selectedChapter.getGplusId());
```

Comment:
You mean keeping a reference to the previous chapter's id or the did you mean caching the whole Fragment somehow. 
I don't think reloading is a problem. We should do the caching better though. It is a different problem.